src/Tabs.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+
+type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (event: React.MouseEvent<HTMLAnchorElement>) => void;
+};
+
+const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  const [tabContent, setTabContent] = useState('');
```

Comment:
The task: 
<img width="963" alt="image" src="https://user-images.githubusercontent.com/99403198/210059792-5e67c29a-151d-4ea3-b3da-bd981ddd25de.png">

Reply:
done